examples/oft-solana/README.md

```diff
@@ -34,6 +36,41 @@ pnpm install
 pnpm test
 ```
 
+### Get Devnet SOL
+
+```bash
+solana airdrop 5 -u devnet
+```
+
+We recommend that you request 5 devnet SOL, which should be sufficient for this walkthrough. For the example here, we will be using Solana Devnet. If you are hitting rate limits after, you can also use the [official Solana faucet](https://faucet.solana.com/).
+
+### Prepare `.env`
+
+```bash
+cp .env.example .env
+```
+
+In the `.env` just created, set `SOLANA_PRIVATE_KEY` to your private key value in base58 format. Since the locally stored keypair is in an integer array format, we'd need to encode it into base58 first. You can create a temporary script called `getBase58Pk.js` in your project root with the following contents:
+
+```js
+import fs from "fs";
+import { Keypair } from "@solana/web3.js";
+import bs58 from "bs58";
+
+const keypairFilePath = `<KEYPAIR_FILE_PATH_HERE>`; // you can view this by running `solana config get`
+
+const data = fs.readFileSync(keypairFilePath, "utf8");
+const keypairJson = JSON.parse(data);
+const keypair = Keypair.fromSecretKey(Uint8Array.from(keypairJson));
+const base58EncodedPrivateKey = bs58.encode(keypair.secretKey);
+
+console.log(base58EncodedPrivateKey);
+```
```

Comment:
Maybe instead we can just add this as a hardhat task?

Not a problem to have this conversion referenced for now, but if we know the exact behaviour we can always add it directly.

Alternatively, @ryandgoulding, I think that this marks an important problem with our Solana DevEx.

We should instead of using the bs58.encode privateKey, use the JSON array format of bs58. The solana-cli uses this format as well, and I think is probably more approachable and consistent with the rest of the solana devex versus having them copy and paste the encoded private key.

Reply:
I vote for the second option. both Solana CLI and Anchor (defaults to) using the keypair in the default location of `~/.config/solana/id.json`. Would be good for us to do the same. Then devs won't need to even bother with this conversion from uint array to base58. We should of course provide the option to reference a different keypair file, just like Anchor.